processing/src/main/java/io/druid/segment/data/GenericIndexed.java

```diff
@@ -167,6 +161,10 @@ public int indexOf(T value)
     bufferIndexed = new BufferIndexed();
   }
 
+  public boolean isAllowReverseLookup() {
```

Comment:
this appears to exist only for the unit tests , can you make it package private?